std/flags/mod.ts

```diff
@@ -36,12 +37,6 @@ function get<T>(obj: { [s: string]: T }, key: string): T | undefined {
   }
 }
 
-function isNumber(x: unknown): boolean {
```

Comment:
I believe you should keep this implementation. This is causing test failures.

`isNumber` here also accept string argument and test if it is a string representation of some number

Reply:
oh,sorry.but i think there is no strict enough if we use string as a number
> in  `std/util/types_test.ts` line 61 , i have  test Hexadecimal
> and i test`0.2e3 0.2e-3 -3.2E3` in chrome'console by `typeof`